src-ui/src/app/components/document-list/document-list.component.ts

```diff
@@ -59,6 +67,7 @@ export class DocumentListComponent implements OnInit, OnDestroy, AfterViewInit {
 
   unmodifiedFilterRules: FilterRule[] = []
 
+  private unsubscribeNotifier: Subject<any> = new Subject()
   private consumptionFinishedSubscription: Subscription
```

Comment:
hey, @shamoon maybe now you can also get rid of this subscription variable.

Reply:
But thats what I use to unsubscribe all? e.g. https://github.com/paperless-ngx/paperless-ngx/blob/e329d72e8b6357ab75332bb3405337664c5f79fa/src-ui/src/app/components/document-list/document-list.component.ts#L172 + https://github.com/paperless-ngx/paperless-ngx/blob/e329d72e8b6357ab75332bb3405337664c5f79fa/src-ui/src/app/components/document-list/document-list.component.ts#L192

Reply:
I mean the `consumptionFinishedSubscription` varaiable 😄

Reply:
Oh haha, yes